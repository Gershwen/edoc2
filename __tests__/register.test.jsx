//snapshot test of /pages/register.js

import { render } from '@testing-library/react';
import Register from '../pages/register';

it('renders registerpage unchanged', () => {
    const { container } = render(<Register />)
    expect(container).toMatchSnapshot()
  })