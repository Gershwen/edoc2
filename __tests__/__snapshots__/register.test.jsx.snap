// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders homepage unchanged 1`] = `
<div>
  <div>
    <div
      class="wrapper"
    >
      <h1>
        eDoc
      </h1>
      <button
        class="sign-in"
        variant="primary"
      >
        Sign in
      </button>
    </div>
    <div
      class="welcome-wrapper"
    >
      <h2>
        Welcome to eDoc
      </h2>
      <p>
        The application built for doctors that takes care of the paperwork so that doctors can take care of the people.
      </p>
    </div>
  </div>
</div>
`;

exports[`renders registerpage unchanged 1`] = `
<div>
  <div>
    <div>
      <h1>
        eDoc
      </h1>
      <form
        class="form-container"
      >
        <div
          class="mb-3"
        >
          <label
            class="form-label"
          >
            Username
          </label>
          <input
            class="form-control"
            id="username"
            name="username"
            placeholder="Enter username"
            type="text"
          />
        </div>
        <div
          class="mb-3"
        >
          <label
            class="form-label"
          >
            Password
          </label>
          <input
            class="form-control"
            id="password"
            name="password"
            placeholder="Password"
            type="password"
          />
        </div>
        <select
          aria-label="Default select example"
          class="form-select"
        >
          <option>
            Open this select menu
          </option>
          <option
            value="admin"
          >
            Administrator
          </option>
          <option
            value="doctor"
          >
            Doctor
          </option>
        </select>
        <div
          class="signUpOrLogIn-container"
        >
          <button
            class="signup-btn btn btn-primary"
            type="submit"
          >
            Sign Up
          </button>
          <a
            href="/login"
          >
            Log in
          </a>
        </div>
      </form>
      <style>
        
            body {
              background: #e8f1ff;
              margin: 0;
              padding: 1em;
            }
            h1,
            h2,
            h3,
            p {
              font-family: "Roboto", sans-serif;
              color: #3b75d9;
            }
          
      </style>
    </div>
  </div>
</div>
`;
